@JRC/bootstrap.m

```diff
@@ -110,18 +110,21 @@ function bootstrap(obj, varargin)
         cfgData.outputDir = workingdir;
     end
 
-    dlgAns = questdlg('Would you like to specify a probe file?', 'Bootstrap', 'No');
-    switch dlgAns
-        case 'Yes' % select .prb file
-            probedir = workingdir;
-            if isempty(dir(fullfile(workingdir, '*.prb')))
-                probedir = fullfile(jrclust.utils.basedir(), 'probes');
-            end
-            [probefile, probedir] = jrclust.utils.selectFile({'*.prb', 'Probe files (*.prb)'; '*.*', 'All Files (*.*)'}, 'Select a probe file', probedir, 0);
-            cfgData.probe_file = fullfile(probedir, probefile);
-
-        case {'Cancel', ''}
-            return;
+    if ~(exist('cfgData','var') && ... % if probe file already specified, no need to ask for it again
+            ~isempty(regexp(cfgData.probe_file,['(?<=\' filesep ')\w+?(?=.prb)'], 'once')))
```

Comment:
Couple of issues with this line:

- throws if you try to bootstrap off of a .bin/.dat file instead of a .meta file
- too restrictive (requires probe file to end in `prb`)